src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1939,58 +1939,85 @@ public abstract partial class DbDataReader : System.MarshalByRefObject, System.C
         public abstract object this[int ordinal] { get; }
         public abstract object this[string name] { get; }
         public abstract int RecordsAffected { get; }
-        public virtual int VisibleFieldCount { get { throw null; } }
+        public virtual int VisibleFieldCount => throw null;
         public virtual void Close() { }
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
         public abstract bool GetBoolean(int ordinal);
+        public bool GetBoolean(string name) => throw null;
         public abstract byte GetByte(int ordinal);
+        public byte GetByte(string name) => throw null;
         public abstract long GetBytes(int ordinal, long dataOffset, byte[] buffer, int bufferOffset, int length);
+        public long GetBytes(string name, long dataOffset, byte[] buffer, int bufferOffset, int length) => throw null;
         public abstract char GetChar(int ordinal);
+        public char GetChar(string name) => throw null;
         public abstract long GetChars(int ordinal, long dataOffset, char[] buffer, int bufferOffset, int length);
+        public long GetChars(string name, long dataOffset, char[] buffer, int bufferOffset, int length) => throw null;
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-        public System.Data.Common.DbDataReader GetData(int ordinal) { throw null; }
+        public System.Data.Common.DbDataReader GetData(int ordinal) => throw null;
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public System.Data.Common.DbDataReader GetData(string name) => throw null;
         public abstract string GetDataTypeName(int ordinal);
+        public string GetDataTypeName(string name) => throw null;
         public abstract System.DateTime GetDateTime(int ordinal);
-        protected virtual System.Data.Common.DbDataReader GetDbDataReader(int ordinal) { throw null; }
+        public System.DateTime GetDateTime(string name) => throw null;
+        protected virtual System.Data.Common.DbDataReader GetDbDataReader(int ordinal) => throw null;
```

Comment:
Which one was removed? `System.DateTime GetDateTime(string name)`? As far as I can see, it's in the implementation and in the final approved API.

Reply:
Hmm, it seemed while reviewing that overloads for GetFieldValueAsync, IsDBNullAsync and GetDbDataReader were still there - maybe you fixed and squashed since then. In any case, it doesn't matter, it's looking good now.

Reply:
Oh, now I see. I think the confusion happened because of the change I did for `=> throw null` which looked as if I added `GetDbDataReader(int ordinal)`. The methods that take an `int` where already there.
Sorry for the confusion.